build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/InternalTestClustersPlugin.java

```diff
@@ -31,6 +32,10 @@ public void apply(Project project) {
         project.getRootProject().getPluginManager().apply(InternalReaperPlugin.class);
         TestClustersPlugin testClustersPlugin = project.getPlugins().apply(TestClustersPlugin.class);
         testClustersPlugin.setRuntimeJava(providerFactory.provider(() -> BuildParams.getRuntimeJavaHome()));
+        testClustersPlugin.setIsReleasedVersion(
+            version -> (version.equals(VersionProperties.getElasticsearchVersion()) && BuildParams.isSnapshotBuild() == false)
+                || BuildParams.getBwcVersions().unreleasedInfo(version) == null
+        );
```

Comment:
:+1:

Reply:
"either it is the current version *and* we're running a release build OR it's not unreleased". That's a lot of double negatives, but I get it.